src/mds/MDSRank.cc

```diff
@@ -2613,11 +2614,31 @@ void MDSRankDispatcher::handle_asok_command(
   int r = 0;
   CachedStackStringStream css;
   bufferlist outbl;
-  if (command == "dump_ops_in_flight" ||
-      command == "ops") {
+  dout(10) << __func__ << ": " << command << dendl;
+  if (command == "dump_ops_in_flight") {
```

Comment:
For the `dump_ops_in_flight` command it will always show the `locks: null` even it have locks, right ? Won't it confusing ? Could we just skip showing the `locks` in this case ?

Reply:
Okay.